api/api.go

```diff
@@ -0,0 +1,64 @@
+package api
+
+import (
+	"bytes"
+	"fmt"
+	"net/http"
+	"os"
+	"github.com/cloud-native-toolkit/itzcli/pkg"
+	"github.com/cloud-native-toolkit/itzcli/pkg/auth"
+	"github.com/gin-gonic/gin"
+	logger "github.com/sirupsen/logrus"
+)
+
+func StartServer() {
+	r := SetUpRouter()
+	r.Run("localhost:8080")
+}
+
+func SetUpRouter() *gin.Engine {
+	gin.SetMode(gin.ReleaseMode)
+	r := gin.New()
+	currentDir, _ := os.Getwd()
+	r.LoadHTMLGlob(fmt.Sprintf("%s/templates/*", currentDir))
+	r.GET("/login", GetTechZoneToken)
+	return r
+}
+
+
+func GetTechZoneToken(c *gin.Context) {
+	// Grab the access token
+	access_token := c.Query("token")
+	if access_token == "" {
+		c.HTML(http.StatusUnauthorized, "error.html", "")
+		logger.Debug("Missing required access token...")
+		auth.ErrorGettingToken()
+		return
+	}
+	// Okay, we have the access token so let's make our API call to TechZone now
+	accessTokenurl := fmt.Sprintf("https://auth.techzone.ibm.com/user?access_token=%s", access_token)
+  
+	TechZoneData, err := pkg.ReadHttpGetTWithFunc(accessTokenurl, "", func(code int) error {
+		logger.Debugf("Handling HTTP return code %d...", code)
+		return nil
+	})
+	if err != nil {
+		c.HTML(http.StatusBadRequest, "error.html", "")
+		auth.ErrorGettingToken()
+		return
+	}
+	// parse the body and grab requestJson
+	jsoner := auth.NewJsonReader()
+	TechZoneDataR := bytes.NewReader(TechZoneData)
```

Comment:
But should this really be uppercase? That may seem nitpicky of me, but there is a method to my madness--someone reading this is going to assume it's a public value and might look outside for the function for it or wonder if this is a side-effect function.